ndbench-core/src/main/java/com/netflix/ndbench/core/DataBackfill.java

```diff
@@ -105,7 +114,7 @@ public Void call() throws Exception {
                         try {
                             String key = "T" + k;
                             String result = backfillOperation.process(client, key);
-                            logger.debug("Backfill Key:" + key + " | Result: " + result);
+                            logger.info("Backfill Key:" + key + " | Result: " + result);
```

Comment:
this log statement is inside a loop inside a thread pool. why raise to info level?

Reply:
This was not intentional, updating it now